### DISCUSSION TITLE
Best way to remove DRM from iTunes m4b files?

### DISCUSSION THREAD

**Discussion by tmr0861** [seanap/Plex-Audiobook-Guide#63] [Q&A]
This is a great guide to get Plex to serve my audiobook library.  I managed to get all of my Audible books drm-free and moved all of my other non-drm books to the library, but I have not found a way to get Plex to be able to stream the books we purchased from iTunes.

Does anyone know of a good tool to do this with?

**Comment by seanap** [comment]
I don't have any direct experience with this so hopefully others can provide more information.  After a quick google search I found this blog on TuneFab's site, https://www.tunefab.com/tutorials/apple-music-drm--removal.html . It mentions that there are freeware and paid software available.  What I would look for with any sort of converter is the ability to retain chapters.  

I haven't used Tunefab, or any other itunes removal software, so definitely do the free trials to test it out first.  Let us know what ends up working for you.

**Reply by tmr0861** [reply]
Thanks for the suggestions.  I had also run across those 2 options, but I was reluctant to try them until I at least reached out to see if there were better options.

Now I have tried Tunefab and it does work, but it is very slow and loses the chapters.  It actually runs iTunes in the background and then must hook into the output stream somehow and encodes the new file.  They claim 10x speed, but it seems much slower than that.  I then tried to use Requiem and unfortunately, I cannot get iTunes 10.7 to authorize with the iTunes store (tried both Windows 11 and an old Windows 10 PC), so I guess I am SoL there too.  

I guess I am stuck using Tunefab and maybe try to get the chapters out from the original m4b file and MediaInfo.  This is just a one-time thing once I get my books onto Plex. From now on, we will be buying our books from Audible and using Libation to get them out of jail.

Thanks for trying!

**Comment by RaneyDazed** [comment]
> This is a great guide to get Plex to serve my audiobook library. I managed to get all of my Audible books drm-free and moved all of my other non-drm books to the library, but I have not found a way to get Plex to be able to stream the books we purchased from iTunes.
> 
> Does anyone know of a good tool to do this with?
Have you looked at inAudible yet? I haven’t tried anything with iTunes in quite a while. Especially audiobooks. Might be a good idea to check it out?

**Reply by tmr0861** [reply]
Thanks for the suggestion.  It looks like inAudible is only for Audible books, but thanks to seanap's guide, I was able to get all my Audible books out of jail using Libation.  Great tool and did a great job!
